modules/otel-collector-components.adoc

```diff
@@ -557,7 +557,38 @@ config: |
 <2> The default exporter when the attribute value is not present in the table in the next section.
 <3> The table that defines which values are to be routed to which exporters.
 
-You can optionally create an `attribute_source` configuratiion, which defines where to look for the attribute in `from_attribute`. The allowed value is `context` to search the context, which includes the HTTP headers, or `resource` to search the resource attributes.
+You can optionally create an `attribute_source` configuration, which defines where to look for the attribute in `from_attribute`. The allowed value is `context` to search the context, which includes the HTTP headers, or `resource` to search the resource attributes.
+
+[id="cumulativetodelta-processor_{context}"]
+=== Cumulative to Delta processor
+
+The Cumulative to Delta processor is currently a link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] feature only.
+
+This processor converts monotonic, cumulative-sum, and histogram metrics to monotonic delta metrics.
+
+[NOTE]
+====
+This processor does not convert non-monotonic sums and exponential histograms.
+====
+
+.OpenTelemetry Collector custom resource with an enabled Cumulative to Delta processor
```

Comment:
```suggestion
.Example of an OpenTelemetry Collector custom resource with an enabled Cumulative to Delta processor
```

Reply:
Thank you for the suggestion. I'm updating this line.